asn1tools/codecs/ber.py

```diff
@@ -1381,9 +1406,21 @@ def encode(self, data, encoded):
 
     def decode(self, data, offset):
         offset = self.decode_tag(data, offset)
-        _, offset = decode_length_definite(data, offset)
+        indefinite = False
+        if data[offset] == 0x80:
+            # Indefinite length field
+            offset = offset + 1
+            indefinite = True
+        else:
+            # Definite length field
+            length, offset = decode_length_definite(data, offset)
 
-        return self.inner.decode(data, offset)
+        values, end_offset = self.inner.decode(data, offset)
+        if indefinite:
+            if data[end_offset:end_offset+2] != EOC_TAG:
+                raise DecodeError('Expected End-Of-Contents tag at offset: {}'.format(end_offset))
```

Comment:
Inconsistency. "End-Of-Contents" was "end-of-contents" in other exception in the same file. Use lower case?